### PR TITLE
[Tier-3] Cephci integration of polarion testcase : CEPH-83575077

### PR DESCRIPTION
**Pull request by anrao19**
# Description
[Tier-3] Cephci integration of polarion testcase : CEPH-83575077
fill workload in versioned bucket with runtime 2hours, with number of objects provided

logs: 
http://magna002.ceph.redhat.com/ceph-qe-logs/anuchaithra/cephci-run-6LH0MX/
http://magna002.ceph.redhat.com/ceph-qe-logs/anuchaithra/cephci-run-84S9K8/ 

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by viduship — COMMENTED** [review]
@anrao19 , have we separated sync and client rgws as part of the baremetal multisite deployments? Please add the support if not.

**Comment by anrao19** [comment]
> @anrao19 , have we separated sync and client rgws as part of the baremetal multisite deployments? Please add the support if not.

@viduship  Added the support

File: suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml [hunk 1]
```diff
@@ -451,6 +451,41 @@ tests:
...
+            drivers:
+              - folio01
+              - folio02
+              - folio03
+            workload_type: fill_runtime
+            run_time: 7200
+            record_sync_on_site: ceph-sec
+            bucket_prefix: cosbench01-bkt-
+            number_of_buckets: 1
+            number_of_objects: 2000000
```
Comment (hmaheswa) [inline suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml:470]: @anrao19 , please add support for object starting number as well
that way we can split the workload between many clusters and trigger a cleanup workload for the particular range of objects

Comment (anrao19) [reply suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml:470]: @hmaheswa for clean up we are taking object count from bucket stats

Comment (hmaheswa) [reply suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml:470]: @anrao19 we can have the facility of splitting the cleanup workload as well
by specifying the objects starting number (and ending number or objects count)

Comment (anrao19) [reply suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml:470]: Changes made @hmaheswa

**Review by ckulal — COMMENTED** [review]
Looks good

File: tests/misc_env/push_cosbench_workload.py [hunk 2]
```diff
@@ -87,7 +88,7 @@
   <workflow>
     <workstage name="preparing_cluster">
         <work type="prepare" workers="1" config="cprefix=bucket_prefix;containers=r(1,bucket_count);
-        objects=r(1,objects_count);sizes=h(1|5|25,5|50|40,50|256|25,256|512|5,512|1024|3,1024|5120|1,5120|51200|1)KB"/>
+        objects=r(obj_num_start,objects_count);sizes=h(1|5|25,5|50|40,50|256|25,256|512|5,512|1024|3,1024|5120|1,5120|51200|1)KB"/>
```
Comment (hmaheswa) [inline tests/misc_env/push_cosbench_workload.py:91]: @anrao19 could you please replace "objects_count" with something similar to "obj_num_end"
and calculate obj_num_end = obj_num_start + objects_count in the below code, and replace obj_num_end in the workload conf

Comment (anrao19) [reply tests/misc_env/push_cosbench_workload.py:91]: @anrao19 , I don't think that will work.

**Review by hmaheswa — APPROVED** [review]

File: tests/misc_env/push_cosbench_workload.py (hunk 2 above)
Comment (hmaheswa) [reply tests/misc_env/push_cosbench_workload.py:91]: okay no problem @anrao19 
let me look into it in the coming PR..

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**